rust/headless-client/src/windows.rs

```diff
@@ -34,6 +31,6 @@ pub(crate) fn notify_service_controller() -> Result<()> {
 }
 
 pub(crate) fn setup_before_connlib() -> Result<()> {
-    wintun_install::ensure_dll()?;
+    firezone_bin_shared::windows::ensure_dll()?;
```

Comment:
Should we do this in `TunDeviceManager::new` instead? It feels like a concern that can be completely hidden behind that API.